variables.tf

```diff
@@ -249,6 +249,12 @@ variable "disable_outbound_traffic_protection" {
   default     = false
 }
 
+variable "operating_system" {
+  type        = string
+  description = "The operating system of the workers in the default worker pool. If no value is specified, RHEL 8 will be used as the default."
```

Comment:
surely it won't always be RHEL 8. I guess it will default to whatever the current default OS is.